app/views/admin/edition_world_tags/edit.html.erb

```diff
@@ -16,34 +17,26 @@
     ) do |form| %>
       <%= form.hidden_field :previous_version %>
 
-      <div class="form-group"
-        data-module="taxonomy-tree-checkboxes"
-        data-content-id="<%= @edition.content_id %>"
-        data-content-format="<%= @edition.content_store_document_type %>"
-        data-content-public-path="<%= public_document_path(@edition) %>">
-
-        <%= render(
-          partial: "/admin/shared/tagging/taxonomy",
-          locals: {
-            selected_taxons: @tag_form.selected_taxons,
-            level_one_taxons: @world_taxonomy.all_world_taxons
-          }
-        ) %>
-      </div>
+      <%= render partial: "/components/miller-columns", locals: {
+        id: "taxonomy_tag_form[taxons]",
+        searchable: true,
+        items: @world_taxonomy.all_world_taxons_transformed(@tag_form.selected_taxons),
+      } %>
 
-      <h2>Selected topics</h2>
-      <div class="content content-bordered hidden" data-module="breadcrumb-preview">
-      </div>
+      <%= render "govuk_publishing_components/components/details", {
+        title: "Changes are applied to a live page as soon as you update the tags"
+      } do %>
+        If this content has already been published and you add new topics, then the last published edition will appear on those topic pages immediately, before you publish a new edition.
+      <% end %>
 
-      <p class="warning">
-        Warning: topic changes to published content appear instantly on the live site.
-      </p>
+      <div class="govuk-button-group govuk-!-margin-top-7">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Save topic changes",
+          name: "save",
+          value: "save",
```

Comment:
do we need this so have a save name and value? I think that's there for the save and continue flow for topics right?

Reply:
Good catch! Have removed them